Extend/ListModel.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Extend;
+
+/**
+ * @inheritdoc
+ */
+class ListModel extends ListModel_parent
+{
+
+    /**
+     * Function for loading the list including inactive items
+     *
+     * @return array
+     */
+    public function getListIncludingInactive()
+    {
+        $oListObject = $this->getBaseObject();
+        $sFieldList = $oListObject->getSelectFields();
+        $sQuery = "select $sFieldList from " . $oListObject->getViewName();
```

Comment:
This is a weird mix of methods for string concatenation. It should be consistent.
```suggestion
        $sQuery = "select {$sFieldList} from {$oListObject->getViewName()}";
```
```suggestion
        $sQuery = 'select ' . $sFieldList . ' from ' . $oListObject->getViewName();
```